composer.json

```diff
@@ -18,8 +18,8 @@
     "bitexpert/slf4psrlog": "^0.1.3"
   },
   "require-dev": {
-    "phpunit/php-code-coverage": "^4.0.1",
-    "phpunit/phpunit": "^5.6",
+    "phpunit/php-code-coverage": "^5.0.0",
```

Comment:
Fair point, we might not need it any more. Did not realize. Thanks for pointing out ;)